.gitignore

```diff
@@ -1,2 +1,3 @@
 /frontend/version
 /api/bulk_preloaded.go
+.DS_Store
```

Comment:
Nit: Put in a separate PR and keep the newline.

Reply:
Will do. Removed commit.